helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/PropertyStoreAccessor.java

```diff
@@ -55,8 +56,8 @@
   public Response getPropertyByPath(@PathParam("clusterId") String clusterId,
       @PathParam("path") String path) {
     path = "/" + path;
-    if (!isPathValid(path)) {
-      LOG.info("The propertyStore path {} is invalid for cluster {}", path, clusterId);
+    if (!ZooKeeperAccessor.isPathValid(path)) {
+      LOG.error("The propertyStore path {} is invalid for cluster {}", path, clusterId);
```

Comment:
Nit, I don't think this error message is appropriate. I remembered @lei-xia reviewed this in the previous ticket, this path invalidation is not an error in the server. So keeping info (or warn) may be good enough.

Reply:
That makes sense. This is not the error from the server's perspective. Will change it back to info.